### DISCUSSION TITLE
Option to reply as the bot in reviews

### DISCUSSION THREAD

**Discussion by getawife** [InfinityBotList/.github#7] [Ideas]
Suggestion:
Add an option that allows the bot developers to reply as the bot to a review.
EG:
User A - Cool bot
Bot - Ty
(not a very good example ik)

**Comment by CodeMeAPixel** [comment]
I guess this is something we can look into! @cheesycod

**Comment by cheesycod** [comment]
HMMM... I suppose, will think about it

**Comment by TyDoesDev** [comment]
# Update 
This is a work in progress @SyntaxErrorSolos
